__tests__/app.test.js

```diff
@@ -110,12 +111,63 @@ describe('/api.articles', () => {
           });
       });
 });
+describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 sends an object containing the article related to the id to the client, ordered by lastest posting', () => {
+        return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({body}) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy('created_at', {
+            descending: true,
+            coerce: true
+          });
+        for (const key in comments) {
+            expect(comments[key]).toHaveProperty("comment_id");
+            expect(comments[key]).toHaveProperty("votes");
+            expect(comments[key]).toHaveProperty("created_at");
+            expect(comments[key]).toHaveProperty("author");
+            expect(comments[key]).toHaveProperty("body");
+            expect(comments[key]).toHaveProperty("article_id");
+          };
+        });
+      });
+    });
+   
+    test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
+        return request(app)
+          .get('/api/articles/999/comments')
+          .expect(404)
+          .then(({body}) => {
+            expect(body.msg).toBe('No article found for article_id: 999');
+          });
+      });
+      test('GET:400 sends an appropriate status and error message when given an invalid id', () => {
+        return request(app)
+          .get('/api/articles/not-an-article/comments')
+          .expect(400)
+          .then(({body}) => {
+            expect(body.msg).toBe('Bad request');
+          });
+    });
+
+
```

Comment:
We should check what happens when the article exists but has no comments associated with it 🙂